app/views/members/voting_members/edit.html.haml

```diff
@@ -0,0 +1,39 @@
+%h1 Voting Member Agreement
+
+%p
+  Filling out this form makes you eligible to become a voting member, but a membership coordinator will still need to confirm
+  that you attended a voting member training and then give you voting member privileges in the app.
+
+%p Please make sure you've read through the following documents before filling out this form:
+
+%ul
+  %li= link_to "DU voting member policies", POLICIES_URL, target: "_blank"
+  %li= link_to "DU voting confidentiality policy", POLICIES_URL, target: "_blank"
+  %li= link_to "DU voting principles and processes", POLICIES_URL, target: "_blank"
```

Comment:
Remove this line, because this stuff will be included in the voting member policies